### ISSUE TITLE
Workflow skipping meta class if methodList of class is empty

### ISSUE THREAD

**Issue by mostobriv** [Vector35/workflow_objc#66]
If `methodList` for `ClassInfo` is empty or `nullptr`, workflow just skip such class completely ignoring metaclass, thus info about classes with only class methods (`+[...]`) not being applied to database.

There is piece of code, responsible for skipping iteration:
https://github.com/Vector35/workflow_objc/blob/3bbf294dba4c46a8e42a426ea4f561a72896a2b6/InfoHandler.cpp#L245-L246

And there is the part that applying info gathered from metaclass:
https://github.com/Vector35/workflow_objc/blob/3bbf294dba4c46a8e42a426ea4f561a72896a2b6/InfoHandler.cpp#L277-L291

I think it should be placed earlier on iteration or mb consider to change response to empty methods list as in the same loop ivar's being applied also, i'm not yet much familiar with objective-c internals, but mb there is some cases when class haven't methods but ivars.

**Comment by mostobriv** [comment]
Here is example of code:
```objc
#import <Foundation/Foundation.h>

@interface UrlConnection : NSObject
@end

@implementation UrlConnection

+(void) load {
  NSLog(@"Loading UrlConnection");
}

+ (BOOL)canHandleRequest:(NSString *)type 
                  forUrl:(NSString *)url {
  NSLog(@"Called canHandleRequest:forUrl:");
  return FALSE;
} 
@end

@interface Kek : NSObject
@end

@implementation Kek

+(void) load {
  NSLog(@"Loading Kek");
}

-(void)wow {
  
}

+ (BOOL)ooooh:(NSString *)type 
                  forKke:(NSString *)url {
  NSLog(@"Called canHandleRequest:forUrl:");
  return FALSE;
} 
@end

int main (int argc, const char * argv[])
{

   [UrlConnection canHandleRequest:@"GET" forUrl:@"https://ya.ru"];
   NSLog (@"Hello, World!");
   return 0; 
} 
```

Recognised functions in BN looks like:
<img width="544" alt="image" src="https://github.com/Vector35/workflow_objc/assets/15954313/68c26b06-1497-44b8-b32a-5cf5579dfdc5">

As `Kek` got instance methods, so its `ClassInfo.methodList` isn't empty and metaclass info is applied, but for `UrlConnection` there no instance methods, but only class methods, so metaclass is skipped.

**Comment by 0cyn** [comment]
Resolved in dev builds >= 4.1.5465
